web/html/src/manager/admin/password-policy/password-policy.tsx

```diff
@@ -0,0 +1,208 @@
+import { hot } from "react-hot-loader/root";
+
+import withPageWrapper from "components/general/with-page-wrapper";
+import { Form, Text, Check } from "components/input";
+import { Panel } from "components/panels/Panel";
+import { TopPanel } from "components/panels/TopPanel";
+import { AsyncButton } from "components/buttons";
+import { showErrorToastr, showSuccessToastr } from "components/toastr/toastr";
+
+// Assume you have an API hook similar to useLifecyclePaygActionsApi
+import usePasswordPolicyApi from "manager/admin/password-policy/use-password-policy-api";
+import { useState } from "react";
+
+type PasswordPolicyProps = {
+  minLength: bigint;
+  maxLength: bigint;
+  digitsFlag: boolean;
+  lowerCharFlag: boolean;
+  upperCharFlag: boolean;
+  consecutiveCharFlag: boolean;
+  specialCharFlag: boolean;
+  specialCharList: string | null;
+  restrictedOccurrenceFlag: boolean;
+  maxCharOccurrence: bigint;
+  isUyuni: boolean;
+};
+
+
+const PasswordPolicy = (props: PasswordPolicyProps) => {
+  const [policy, setPolicy] = useState({
+    properties: {
+      minLength: props.minLength.toString(),
+      maxLength: props.maxLength.toString(),
+      digitsFlag: props.digitsFlag,
+      lowerCharFlag: props.lowerCharFlag,
+      upperCharFlag: props.upperCharFlag,
+      consecutiveCharFlag: props.consecutiveCharFlag,
+      specialCharFlag: props.specialCharFlag,
+      specialCharList: props.specialCharList || "",
+      restrictedOccurrenceFlag: props.restrictedOccurrenceFlag,
+      maxCharOccurrence: props.maxCharOccurrence.toString(),
+      isUyuni: props.isUyuni,
+    },
+    errors: {},
+  });
+
+  const { updatePolicy } = usePasswordPolicyApi();
+
+  return (
+    <TopPanel
+      title={t(policy.properties.isUyuni ? "SUSE Manager" : "Uyuni").concat(" Configuration - Password Policy")}
```

Comment:
This won't work for our translators, because the order of the words will be different in different languages. It's a bit more verbose, but the below would be preferred from the translation perspective:  

```tsx
title={policy.properties.isUyuni ? t("...") : t("...")}
```

Alternatively, we could sidestep the whole problem by not including the product name here at all, e.g.

```tsx
title={t("Password Policy Configuration")}
```

Personally I prefer the second approach, but I don't have a strong preference.

Reply:
Agreed with the 2nd approach